javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
You should only do this line of code if there was something truthy stored in `localStorage`. If you have your code like this and it's the first time a user visits the website, the array in `todos` will be replaced with `null`, since there won't be anything in `localStorage` for the first visit, which will entirely break the application for that user _forever_ 😭